src/public/js/comment.js

```diff
@@ -0,0 +1,56 @@
+(function ($) {
+    var newCommentForm = $('#addComment');
+        newNameInput = $('#commenterName'),
+        newCommentInput = $('#comment'),
+        errorDiv = $('#error')
+        commentArea = $('#comment-area');
+
+    newCommentForm.submit(function (event) {
+        event.preventDefault();
+
+        var newName = newNameInput.val();
+
+        try{
+            newName = checkPersonName(newName);
+        } catch(e) {
+            $("#diverror").html("Error: Commenter Name must be provided");
```

Comment:
Can you return after you create a diverror element? This way the next statement is not followed through when an error occurs.